tasks/apt.yml

```diff
@@ -65,6 +65,7 @@
       - libproj-dev
       - libgeos-dev
       - libspatialite-dev
+      - openssh-client
```

Comment:
this should be added in the system dependencies, because if spatialite is not being used (one could use PostgreSQL or MySQL)